Classes/WPLegacyEditorViewController.m

```diff
@@ -6,6 +6,41 @@
 CGFloat const WPLegacyEPVCTextViewOffset = 10.0;
 CGFloat const WPLegacyEPVCToolbarHeight = 44.0;
 
+@interface WPLegacyWrapperViewForInputView: UIView
+    @property (nonatomic, strong) UIToolbar *toolbar;
+@end
+
+@implementation WPLegacyWrapperViewForInputView
+
+- (instancetype)initWithToolbar:(UIToolbar *)toolbar {
+    self = [super initWithFrame:CGRectMake(0, 0, self.frame.size.width, toolbar.frame.size.height)];
+    if (self) {
+        _toolbar = toolbar;
+        self.autoresizingMask = UIViewAutoresizingFlexibleHeight;
+        self.backgroundColor = [[WPLegacyEditorFormatToolbar appearance] backgroundColor];
```

Comment:
Should we just use the background colour of the toolbar that's passed into this method?